web/src/components/ServiceList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Card, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        maxWidth: 400,
+        marginLeft: 10,
+        marginTop: 10,
+    },
+    title: {
+        fontSize: 14,
+    },
+    details: {
+        margin: '0 2px',
+    },
+    cardView: {
+        padding: theme.spacing(2),
+    },
+}));
+
+const ServiceList = () => {
+    const [services, setServices] = useState(null);
+    const classes = useStyles();
+    const fetchData = async () => {
+        const response = await axios.get('http://localhost:5000/services/');
```

Comment:
the base url, i.e., `http://localhost:5000/` should be an environment variable. 
so you could do something like `{env.API_HOST}/services`

Reply:
already made a PR about this since will need to be changed in a couple of places

Reply:
scratch that just adding to this PR